src/LeanCode.Contracts/SubscriptionEnvelope.cs

```diff
@@ -1,8 +1,10 @@
+using System.Text.Json;
+
 namespace LeanCode.Contracts;
 
 public sealed class SubscriptionEnvelope
 {
     public Guid Id { get; set; }
     public string TopicType { get; set; } = default!;
-    public string Topic { get; set; } = default!;
+    public JsonDocument Topic { get; set; } = default!;
```

Comment:
I have tested this using TypeScript and there it is equivalent to:
```
interface SubscriptionEnvelope {
  Id: string;
  TopicType: string;
  Topic: object;
}
```
Did not test it using Dart, but I suppose it should be similar.

Reply:
The problem wasn't dart/typescript here, but C#. We will be able to send an object no problem. Are you able to deserialize `JsonDocument` to appropriate topics?

Reply:
Yes, this works